src/collections/collection.ts

```diff
@@ -167,25 +171,35 @@ export class Collection {
     return;//TODOV3 returning as succeeded for now for testing
   }
 
-  find(query: any, projection?: any, options?: any, cb?: any) {
+  find(query: any, options?: any, cb?: any) {
     ({ options, cb } = setOptionsAndCb(options, cb));
-    const cursor = new FindCursor(this, query, projection, options);
+    const cursor = new FindCursor(this, query, options);
     if (cb) {
       return cb(undefined, cursor);
     }
     return cursor;
   }
 
-  async findOne(query: any, projection?: any, options?: any, cb?: any) {
```

Comment:
I think we should make this change in the `findOne` function in `src/driver/collection.ts` as well?